.github/workflows/citations.yml

```
@@ -0,0 +1,19 @@
+name: Update Citations
+on:
+  schedule:
+    - cron: '0 0 * * 1'  # Weekly on Mondays
+  workflow_dispatch:  
+jobs:
+  update:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v5
+        with:
+          python-version: '3.10'
```

Would this work with a later version of Python, it would be great if it could align with the versions BrainGlobe supports (which are according to [SPEC 0](https://scientific-python.org/specs/spec-0000/))
```suggestion
          python-version: '3.13'
```

---

sure, it will work